spark-icons/src/main/res-iconography/contact/drawable/spark_icons_send_message_padded.xml

```diff
@@ -0,0 +1,6 @@
+<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
+    <item
+        android:drawable="@drawable/design_ic_send_message_16dp"
```

Comment:
Some resources have references to old resources, we should fix that:

```
   > Android resource linking failed
     ERROR:/home/runner/work/spark-android/spark-android/spark-icons/build/intermediates/packaged_res/release/drawable/spark_icons_send_message_padded.xml:4: AAPT: error: resource drawable/design_ic_send_message_16dp (aka com.adevinta.spark.sample:drawable/design_ic_send_message_16dp) not found.
```

Reply:
Yep done in 1288018